app/routes/public/sessions.js

```diff
@@ -197,8 +197,10 @@ export default class SessionsRoute extends Route {
         }
       });
     }
-
+    const { success, token } = await this.loader.load(`/events/${eventDetails.id}/chat-token`);
```

Comment:
Wrong logic, first this is duplication of code. Secondly, this should be fetched in the component. That's why it is logging in user before opening the chat